synapse/storage/databases/main/events_bg_updates.py

```diff
@@ -178,9 +176,7 @@ def reindex_txn(txn):
 
             sql = "UPDATE events SET sender = ?, contains_url = ? WHERE event_id = ?"
 
-            for index in range(0, len(update_rows), INSERT_CLUMP_SIZE):
-                clump = update_rows[index : index + INSERT_CLUMP_SIZE]
-                txn.executemany(sql, clump)
```

Comment:
After this change, this will essentially end up as `txn.executemany(sql, update_rows)` for SQLite. Is that OK or should we do this clumping inside of our `execute_batch` method for SQLite?

Reply:
I think the clumping is a bit spurious, I believe `executemany` is fine handling large lists.

Reply:
The docs don't say anything about trying to limit the length of the iterator used, so... 🤷